assets/style.css

```
@@ -356,7 +358,7 @@ body {
 }
 
 .sidebar .menu {
-  padding: 2rem 3rem;
+  padding: 2rem 4rem;
```

Let's replace this padding with just 2rem and then give menu a flex and let it justify content to center, because the icons are breaking on my 13'' screen.

```suggestion
  padding: 2rem;
  display: flex;
  justify-content: center;
```